Chore: adding NRDOT Section (#20202)
newrelic/docs-website: src/content/docs/opentelemetry/nrdot/nrdot-collector.mdx

---
title: New Relic Distributions of the OpenTelemetry Collector
tags:
  - OpenTelemetry
  - OpenTelemetry Collector
  - nrdot-collector-host
  - nrdot-collector-k8s
  - NRDOT
metaDescription: New Relic-supported pre-built distributions of the OpenTelemetry Collector
freshnessValidatedDate: 2024-05-14
---

The New Relic Distribution of OpenTelemetry Collector (NRDOT) is a collection of New Relic-supported pre-built distributions of the OpenTelemetry [Collector](https://opentelemetry.io/docs/collector/).
NRDOT is designed to work seamlessly with New Relic products and services, and it is the recommended way to collect, process, and export open telemetry data for use with New Relic.
Each distribution comes bundled with configs that are pre-configured to integrate with New Relic ingest.

Rather than providing an all encompassing single distribution of the OpenTelemetry Collector, we have focused on providing distributions that are:
1. Lightweight
2. Limited in scope
3. Tailored to specific New Relic experiences

> 4. Pre-configured to report to New Relic's OTLP endpoint

Reviewer: And have additional support?
```suggestion
4. Pre-configured to report to New Relic's OTLP endpoint
5. Supported
```

This allows us to provide a more focused and tested experience for our users, while also allowing us to iterate and improve on these distributions more quickly.
Each distribution is tailored to a specific experience in New Relic and meant to allow users to quickly get started with OpenTelemetry and New Relic.
Our intent is that there be minimal overlap between the distributions and that they can be used in conjunction with other collectors if need be.
Additionally, since they are built using the [OCB](https://opentelemetry.io/docs/collector/custom-collector/), advanced users can build their own distributions if they must include additional components.

<Callout variant="tip">

>   There are scenarios where you may want to deploy the collector in a less opinionated way, we recommend reading our documentation on [collector processing](/docs/opentelemetry/get-started/collector-processing/opentelemetry-collector-processing-intro) to understand how to configure the collector for your specific use case.

Reviewer: Ugh the "Collector for data processing" page stinks and recommending it as hard to stomach. Maybe its the best we can do for the moment, but we really should revisit the content on that page.

Reviewer 2: @cbehera-newrelic Create a ticket to track @jack-berg 's "ugh" page please and add it to the backlog.

</Callout>

Before diving into one of our distributions we recommend familiarizing yourself with the basic concepts of the OpenTelemetry Collector in order to understand how our configurations are structured and how to customize them to your needs.

> ## Deployments [#deployments]

Reviewer: Let's split this out as a column in the table below to reinforce this point. I.e.:

| Distribution | Descrpition | Deployment strategy |
|---|---|---|

---
title: New Relic Distributions of the OpenTelemetry Collector
tags:
  - OpenTelemetry
  - OpenTelemetry Collector
  - nrdot-collector-host
  - nrdot-collector-k8s
  - NRDOT
metaDescription: New Relic-supported pre-built distributions of the OpenTelemetry Collector
freshnessValidatedDate: 2024-05-14
---

> The New Relic Distribution of OpenTelemetry Collector (NRDOT) is a collection of New Relic-supported pre-built distributions of the OpenTelemetry [Collector](https://opentelemetry.io/docs/collector/).

Reviewer 2: ```suggestion
The New Relic Distribution of OpenTelemetry Collector (NRDOT) is a collection of prebuilt distributions of the OpenTelemetry [Collector](https://opentelemetry.io/docs/collector/) supported by New Relic.
```

Our intent is that there be minimal overlap between the distributions and that they can be used in conjunction with other collectors if need be.
Additionally, since they are built using the [OCB](https://opentelemetry.io/docs/collector/custom-collector/), advanced users can build their own distributions if they must include additional components.

<Callout variant="tip">
  There are scenarios where you may want to deploy the collector in a less opinionated way, we recommend reading our documentation on [collector processing](/docs/opentelemetry/get-started/collector-processing/opentelemetry-collector-processing-intro) to understand how to configure the collector for your specific use case.
</Callout>

Before diving into one of our distributions we recommend familiarizing yourself with the basic concepts of the OpenTelemetry Collector in order to understand how our configurations are structured and how to customize them to your needs.

## Deployments [#deployments]

The collector itself is a single binary that can be deployed in a variety of ways.  The typical deployment strategies are:

> - As an Agent: Deployed on the same host as the application.

Reviewer 2: ```suggestion
- As an agent: Deployed on the same host as the application.
```

> - As a Gateway: Deployed as a central point for telemetry collection.

Reviewer 2: ```suggestion
- As a gateway: Deployed as a central point for telemetry collection.
```
